helpers.py

```diff
@@ -59,6 +59,9 @@ def emtr_param_func(params):
     
     reform_output = emtr_with_income_choice(
         emtr_param_func, reform_params, reform_income_choice)
+    
+    sq_output['net_income'] = sq_output['net_income'] *52
```

Comment:
my only suggestion would be to make a new column 'annual_net_income'  and plot that.  ONe reason is that Ben's code multiplies net_income by 52 again.